cypress/integration/Change Password Tests/changePassword.specs.js

```diff
@@ -0,0 +1,43 @@
+ /// <reference types="cypress" />
```

Comment:
Please add an empty line after this.
```suggestion
 /// <reference types="cypress" />
 
```